t/22-dashboard.t

```diff
@@ -123,8 +131,8 @@ $opensuse_test_group->update({parent_id => $test_parent->id});
 $opensuse_group->jobs->find({BUILD => '0048@0815'})->comments->create({text => 'poo#1234', user_id => 99901});
 
 $t->get_ok('/dashboard_build_results?limit_builds=20&show_tags=1')->status_is(200);
-@h2 = $t->tx->res->dom->find('h2 a')->map('text')->each;
-is_deeply(\@h2, ['Test parent'], 'only parent shown, no more top-level job groups');
+@h2 = $t->tx->res->dom->find('h2 a')->map(sub ($e) { $e->text . ($e->attr('title') // '') })->each;
+is_deeply(\@h2, ['Test parent', 'Shows the latest test results for all job groups within this parent job group'], 'only link to parent (and related overview) shown, no more top-level job groups');
```

Comment:
Maybe it makes sense to store the long string in a variable, e.g. `my $test_overview_tooltip = …` and use it in both occurrences.

Reply:
done